docs/config/services/deployments/production.js

```diff
@@ -13,7 +13,7 @@ var cdnUrl = googleCdnUrl + versionInfo.cdnVersion;
 // The currentVersion may not be available on the cdn (e.g. if built locally, or hasn't been pushed
 // yet). This will lead to a 404, but this is preferable to loading a version with which the example
 // might not work (possibly in subtle ways).
-var examplesCdnUrl = versionInfo.isSnapshot ?
+var examplesCdnUrl = versionInfo.currentVersion.isSnapshot ?
   (angularCodeUrl + 'snapshot') :
   (googleCdnUrl + (versionInfo.version || versionInfo.currentVersion));
```

Comment:
You are right. I couldn't find where `version` may have been defined, so I left it untouched in order to avoid breaking anything. But it is clear now that (a) `versionInfo` can never have a `version` property and (b) it is referring to the `version` property of the `SemVer` object (returned from `semver.parse(...)`) of `versionInfo.currentVersion`.